File: cms_web/src/Services/Entities/DepartmentEntity.ts
import UserEntity from "./UserEntity";

interface DepartmentEntity {
  dept_pk?: number;
  dept_code?: number;
  dept_name?: number;
  notes?: string;
  is_active?: "y" | "n";
  encoded_at?: Date;
  encoder_pk?: string;

  user?: UserEntity;
}

export interface DepartmentTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<DepartmentEntity>;
}

export default DepartmentEntity;

File: cms_server/cms_server/Entities/DepartmentEntity.cs
using pos_server.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace cms_server.Entities
{
    public class DepartmentEntity
    {
        public int? dept_pk { get; set; }
        public string dept_code { get; set; }
        public string dept_name { get; set; }
        public string notes { get; set; }
        public string is_active { get; set; }
        public TimeSpan cut_off_start { get; set; }
        public TimeSpan cut_off_end { get; set; }
        public string is_accept_consult { get; set; }
        public DateTime? encoded_at { get; set; }
        public string encoder_pk { get; set; }

        public UserEntity user { get; set; }
    }
}
